exporter/collector/integrationtest/testdata/fixtures/logs/logs_apache_access_batches_expected.json

```diff
@@ -397,25 +447,13 @@
               "zone": ""
             }
           },
-          "textPayload": "127.0.0.2 - - [26/Apr/2022:22:54:38 +0800] \"GET / HTTP/1.1\" 200 4429",
+          "textPayload": "127.0.0.1 - - [26/Apr/2022:22:54:43 +0800] \"-\" 408 -",
```

Comment:
Is this diff because the new service labels changed the cutoff for where the batches get split by bytes?

Reply:
Yeah good catch. I updated the batch size for this fixture and that looks like it got rid of the changes in a new commit https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/613/commits/3bb133f13c6df313788c88948cb3de7f2db8106e

The change against main still looks a little weird but I'm pretty sure that's just how git represented each entry getting shifted down a line: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/613/files#diff-55654849941a179a128e04a60faef5b295c8b07a7acd771f12b3c5dc3265a77e